decoration.sk

```diff
@@ -123,9 +123,9 @@ decoratives before flattening:
 	{attached} torch¦es = minecraft:torch
 	chest¦s = minecraft:chest
 	(wet|moist|hydrated) farmland = minecraft:farmland {Damage:7}
-	unlit furnace¦s = minecraft:furnace
+	[unlit] furnace¦s = minecraft:furnace
 	lit furnace¦s = minecraft:lit_furnace
-	[any] furnace¦s = unlit furnace, lit furnace
+	any furnace¦s = unlit furnace, lit furnace
```

Comment:
The thing with these furnace things is `if player's target block is furnace` should return true if they're looking at a burning furnace, and with the aliases like this, I don't think that'd be the case. But maybe I'm wrong, and in that case, they should write `if player's target block is any furnace`? I can see the argument for both.

Reply:
Oh, I see what you mean.

give player furnace
versus
give player unlit furnace

I’ll probably revert this one for now

Reply:
There's an argument for both. I do see what you mean, `give player furnace` should give an unlit furnace without having to say so. It's probably fine as-is. It's hard to make the call as to which is "better".